evap/staff/management/commands/run_tasks.py

```diff
@@ -26,12 +29,23 @@ def update_courses(self):
                     course.evaluation_end()
                     if course.is_fully_reviewed():
                         course.review_finished()
+                        if not course.is_graded:
+                            course.publish()
+                            evaluation_results_courses.append(course)
+                        elif course.final_grade_documents.exists():
+                            course.publish()
+                            grade_document_courses.append(course)
+                            evaluation_results_courses.append(course)
+                    elif course.final_grade_documents.exists():
+                        grade_document_courses.append(course)
                     course.save()
             except Exception:
                 pass
 
         if courses_new_in_evaluation:
             EmailTemplate.get_evaluation_started_template().send_to_users_in_courses(courses_new_in_evaluation, ['all_participants'])
```

Comment:
please bring this in line with the style one line below (i.e. one method that gets a list of courses)